src/AutoRest.CodeModel/StringExtensions.cs

```diff
@@ -21,12 +21,5 @@ public static IEnumerable<string> ToLines(this string value, bool removeEmptyLin
                 yield return line;
             }
         }
-
-        //https://stackoverflow.com/a/8809437/294804
-        public static string ReplaceFirst(this string text, string oldValue, string newValue)
-        {
```

Comment:
This method is no longer used here, moved it to  `src/AutoRest.CSharp/Common/Utilities/StringExtensions.cs`